addons/l10n_es_edi_sii/views/res_config_settings_views.xml

```diff
@@ -24,12 +24,28 @@
                                   groups="base.group_multi_company"/>
                             <div class="content-group">
                                 <div class="mt16">
+                                    <label for="l10n_es_edi_tax_agency"/>
                                     <field name="l10n_es_edi_tax_agency"/>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_tax_agency', '!=', False)]}">
+                                        No tax agency selected: SII not activated.
+                                    </div>
+                                    <div class="text-muted" attrs="{'invisible': [('l10n_es_edi_tax_agency', '=', False)]}">
+                                        Tax agency selected: SII is activated.
+                                    </div>
```

Comment:
```suggestion
```
It still depends on whether it is activated on the journal.  I would not tell anything here when it has a value.